vizro-core/changelog.d/20230913_134437_alexey_snigir_check_spelling_hook.md

```diff
@@ -0,0 +1,41 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Removed
+
+- A bullet item for the Removed category.
+
+-->
+
+### Added
+
+- Pre-commit `check-spelling` hook for words `colour` and `visualisation`.
```

Comment:
I think this is also not a change we need to notify our users about.

Reply:
Deleted